gatekeeper/manifest.json

```
@@ -0,0 +1,39 @@
+{
+  "display_name": "Gatekeeper",
+  "maintainer": "ara.pulido@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "gatekeeper",
+  "metric_prefix": "gatekeeper.",
+  "metric_to_check": "gatekeeper.constraints",
+  "creates_events": false,
+  "short_description": "Gatekeeper integration",
+  "guid": "a68c72bd-a16a-4fcf-8911-43575dd722b9",
+  "support": "contrib",
+  "supported_os": [
+    "linux"
+  ],
+  "public_title": "Datadog Gatekeeper Integration",
```

```suggestion
  "public_title": "Gatekeeper",
```

I just synced this to staging and the tile is labelled `Datadog Gatekeeper Integration` which is a bit awkward.